latest/ug/automode/create-node-pool.adoc

```diff
@@ -177,3 +177,18 @@ spec:
 * {eam} only supports Linux
 ** `node.kubernetes.io/windows-build`
 ** `kubernetes.io/os`
+
+== Disruption
+
+You can configure EKS Auto Mode to disrupt Nodes through your NodePool in multiple ways. You can use `spec.disruption.consolidationPolicy`, `spec.disruption.consolidateAfter`, or `spec.template.spec.expireAfter`. You can also rate limit EKS Auto Mode's disruption through the NodePool’s spec.disruption.budgets. For instructions on configuring this behavior, see https://karpenter.sh/docs/concepts/disruption/[Disruption] in the Karpenter Documentation. 
```

Comment:
Maybe for "You can also rate limit EKS Auto Mode's disruption" it should be "You can also control the time windows and number of simultaneous Nodes disrupted"

Missing the ticks around `spec.disruption.budgets`

Reply:
Thanks Todd!